examples/nivisa_dmm_measurement/teststand_fixture.py

```diff
@@ -1,13 +1,12 @@
 """Functions to set up and tear down NI-VISA DMM sessions in NI TestStand."""
 
-from _helpers import GrpcChannelPoolHelper, PinMapClient
-from _visa_helpers import (
-    INSTRUMENT_TYPE_DMM_SIMULATOR,
-    create_visa_resource_manager,
-    create_visa_session,
-    log_instrument_id,
-    reset_instrument,
-)
+from _helpers import GrpcChannelPoolHelper
```

Comment:
Can you revert this change? [PEP8 ](https://peps.python.org/pep-0008/#imports)has no problem with importing multiple objects from the same file. I think this will also fix the PR build.

Reply:
I meant to highlight lines 3-9.

Reply:
@dixonjoel, you may need to update the version of ni-python-styleguide in the lock file / your venv (0.4.0 keeps these together now)

Reply:
I updated the pyproject.toml to have >=0.4.0 for ni-python-styleguide. This required me to run poetry lock, so please check the lock file and pyproject file.

I have Poetry (version 1.3.1). I can't remember the correct version, but I remember @mshafer-NI saying I had to have a certain version to get the .lock file right.

Reply:
Also, I'm getting a bunch of D417 errors for docstrings. I'm not sure why since the methods mostly have documentation on them. Unless I need to document the 'self' argument or the docs are formatted incorrectly.
> ./examples/niswitch_control_relays/teststand_fixture.py:[10](https://github.com/ni/measurementlink-python/actions/runs/4098613934/jobs/7067851096#step:9:11):1: D417 Missing argument descriptions in the docstring
./examples/niswitch_control_relays/_helpers.py:52:1: D417 Missing argument descriptions in the docstring
./examples/niswitch_control_relays/_helpers.py:100:1: D417 Missing argument descriptions in the docstring
./examples/nivisa_dmm_measurement/teststand_fixture.py:19:1: D417 Missing argument descriptions in the docstring

Reply:
maybe related?
https://github.com/PyCQA/pydocstyle/issues/443

> I have Poetry (version 1.3.1). I can't remember the correct version, but I remember @mshafer-NI saying I had to have a certain version to get the .lock file right.

@bkeryan this project is using Poety 1.2, right?

Reply:
I think the D417 errors occur because we're mixing numpy and Google style.

numpy uses dashes with no colon: https://numpydoc.readthedocs.io/en/latest/format.html
```
Parameters
----------
x : type
    Description of parameter `x`.
y
    Description of parameter `y` (with type not specified).
```

Google style uses a colon with no dashes: https://google.github.io/styleguide/pyguide.html
```
    """Connects to the next available port.

    Args:
      minimum: A port value greater or equal to 1024.

    Returns:
      The new minimum port.

    Raises:
      ConnectionError: If no available port is found.
    """
```

Reply:
I think you should split the ni-python-styleguide update into a separate PR.

Reply:
I'm fine with that. That just means these lines will have to be separate imports until we migrate the style guide.

Reply:
You don't have to revert the imports. `ni-python-styleguide lint` never required these imports to be on separate lines.